Probesys/bileto
# imp: Allow to choose locale when creating/editing a user

## Related issue(s)



https://github.com/Probesys/bileto/issues/530
https://github.com/Probesys/bileto/issues/551

## How to test manually



1. in the admin, create a new user
2. verify that it works and that you're redirected to their page
3. edit the user
4. verify that it works
5. go to the edit page of a user managed by LDAP (e.g. charlie@example.com)
6. verify that an alert informs that some fields cannot be changed, and that email and name are disabled, while the password field isn't shown

## Checklist
